### ISSUE TITLE
Commit 8b899fe (GP-4957: Add debugger launchers for gdb on Windows) breaks Linux qemu-user debugging

### ISSUE THREAD

**Issue by electricworry** [NationalSecurityAgency/ghidra#7095]
**Describe the bug**
Until this commit, one could perform qemu-user debugging from Ghidra using the existing qemu-gdb.sh script. After this commit it does not work.

**To Reproduce**
1. Import an i386 ELF to Ghidra project
2. Open it in Debugger tool
3. Select Debugger > Configure and launch using > qemu+gdb
4. Parameters:
    - Image: path to ELF
    - Arguments: NONE
    - QEMU command: qemu-i386
    - QEMU Port: 1234
    - Extra qemu arguments: NONE
    - gdb command: gdb-multiarch
5. Fails to initialise. Terminal outputs `localhost:1234: Connection timed out.` It would appear that the command to run the qemu emulator fails (hence the inability to connect to the gdbserver port); see below.

**Expected behavior**
It should run

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04.3
 - Java Version: OpenJDK 21.0.3
 - Ghidra Version: aff12cd6e058c32636fb7859bafb4267bef7f1b5
 - Ghidra Origin: locally built

**Additional context**
It appears the regression is in qemu-gdb.sh. Before:
```
if [ -z "$TTY_TARGET" ]
then
  "$GHIDRA_LANG_EXTTOOL_qemu" $OPT_EXTRA_QEMU_ARGS $@ &
else
  "$GHIDRA_LANG_EXTTOOL_qemu" $OPT_EXTRA_QEMU_ARGS $@ <$TTY_TARGET >$TTY_TARGET 2>&1 &
fi
```

After:
```
if [ -z "$TTY_TARGET" ]
then
  "$GHIDRA_LANG_EXTTOOL_qemu" -gdb tcp::$QEMU_GDB -S $OPT_EXTRA_QEMU_ARGS $@ &
else
  "$GHIDRA_LANG_EXTTOOL_qemu" -gdb tcp::$QEMU_GDB -S $OPT_EXTRA_QEMU_ARGS $@ <$TTY_TARGET >$TTY_TARGET 2>&1 &
fi
```

The qemu switches `-gdb` and `-S` are for qemu-system emulation and incompatible with qemu-user. Has Ghidra quietly dropped support for qemu-user to support qemu-system instead?

**Comment by nsadeveloper789** [comment]
Oops. Okay, so no, we did not intend to quietly drop support for qemu-user. Just an honest mistake and we overlooked this particular launcher in testing the change. I saw `QEMU_GDB` declared as an option at the top, but it was not getting used in the script. Must have invoked the wrong qemu was asking for help. All that said, you have a few options:

1. Just use the "gdb remote" launcher and connect it to qemu-user manually.
2. Revert the two lines you pointed out in the launch script.
3. Correct the lines to use the equivalent options for qemu-user.

If you pick option 3, we'd certainly be interested in a pull request. I've put an internal ticket in, as I think we should have two qemu launch scripts. One for user and one for system.
